discount-function-metafields-block/README.md.liquid

```diff
@@ -0,0 +1,5 @@
+# Block Extension
+
```

Comment:
...using the "admin.discount-details.function-metafields-render" extension block target and allows merchants to configure discount function metafields...